__tests__/app.test.js

```diff
@@ -189,60 +189,105 @@ describe("GET /api/articles/:article_id/comments", () => {
+  test("POST: 201 inserts a new comment to the db and sends the new comment back to the client.", () => {
     const newComment = {
       username: "icellusedkars",
-      body: "Ik hou van Gouda kaas!"
-    }
+      body: "Ik hou van Gouda kaas!",
+    };
 
     return request(app)
-    .post('/api/articles/1/comments')
-    .send(newComment)
-    .expect(201)
-    .then((response) => {
-      const {body} = response;
-      expect(body.comment.author).toBe("icellusedkars")
-      expect(body.comment.body).toBe("Ik hou van Gouda kaas!")
-    })
-  })
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const { body } = response;
+        expect(body.comment.author).toBe("icellusedkars");
+        expect(body.comment.body).toBe("Ik hou van Gouda kaas!");
+      });
+  });
 
-  test('Should return an error if the username doesnt exist', () => {
+  test("Should return an error if the username doesnt exist", () => {
     const newComment = {
       username: "Kerstman159",
-      body: "Hallo!"
-    }
+      body: "Hallo!",
+    };
 
     return request(app)
-    .post('/api/articles/1/comments')
-    .send(newComment)
-    .expect(400)
-    .then((response) => {
-      const { body } = response;
-      const {msg} = body;
-      expect(msg).toBe(("Bad request"))
-    })
-  })
-
-  test('POST: 400 responds with a status and error message when provided with a bad comment, for example no username!', () => {
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        const { msg } = body;
+        expect(msg).toBe("Bad request");
+      });
+  });
 
+  test("POST: 400 responds with a status and error message when provided with a bad comment, for example no username!", () => {
     const newComment = {
-      body: "Thats cool!"
-    }
+      body: "Thats cool!",
+    };
 
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        const { body } = response;
+        const { msg } = body;
+
+        expect(msg).toBe("Bad request");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("Updates the votes and returns the updated article.", () => {
+    const votes = { votes: 2 };
 
     return request(app)
-    .post('/api/articles/1/comments')
-    .send(newComment)
-    .expect(400)
-    .then((response) => {
-      const {body} = response
-      const {msg} = body;
+      .patch("/api/articles/1")
+      .send(votes)
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const { article } = body;
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("votes", 102);
+      });
+  });
+
+  test("PATCH: 400 responds with error if votes is missing or invalid", () => {
```

Comment:
In your description you've got "responds with error if votes is missing OR invalid" but then you only test for an invalid votes object (with a cool string input 😎). If votes is missing, I would say that's a different behaviour that still needs to be tested!